dashboard/test/ui/features/applab/dataBlocks.feature

```diff
@@ -0,0 +1,13 @@
+@no_ie
+@dashboard_db_access
+Feature: App Lab Data Blocks
+
+  Background:
+    Given I sign in as a student
+    And I am on "http://studio.code.org/s/allthethings/stage/18/puzzle/8?noautoplay=true"
```

Comment:
It tests the app lab data blocks, of course :) I've added a more descriptive comment.